src/geos.h

```diff
@@ -100,6 +102,17 @@ enum {
       PyErr_SetString(PyExc_ValueError,                                                  \
                       "A linearring requires at least 4 coordinates.");                  \
       break;                                                                             \
+    case PGERR_NAN_COORD:                                                                \
+      PyErr_SetString(PyExc_ValueError,                                                  \
+                      "A NaN, Inf or -Inf coordinate was supplied. Remove the "          \
+                      "coordinate or adapt the 'handle_nans' parameter.");               \
+      break;                                                                             \
+    case PGWARN_NAN_COORD:                                                               \
```

Comment:
This doesn't seem to be used anymore. Idea initially was to also have the option to raise a warning instead of error?